app/models/metric/rollup.rb

```diff
@@ -239,14 +240,15 @@ def self.rollup_realtime_perfs(obj, rt_perfs, new_perf = {})
         new_perf_counts[col] ||= 0
 
         value = rt.send(col)
-
-        if obj.kind_of?(VmOrTemplate) && BURST_COLS.include?(col)
-          new_perf[:min_max] ||= {}
-          rollup_burst(col, new_perf[:min_max], rt.timestamp, value)
-        end
-
         Metric::Aggregation::Aggregate.column(col, nil, new_perf, new_perf_counts, value)
       end
+
+      next unless obj.kind_of?(VmOrTemplate)
+      new_perf[:min_max] ||= {}
+      BURST_COLS.each do |col|
+        value = rt.send(col)
+        rollup_burst(col, new_perf[:min_max], rt.timestamp, value)
+      end
```

Comment:
What did moving this section out of the loop above help with?

Reply:
It was embedded in the loop of ` Metric::Capture.capture_cols.each do |col|` which wouldn't cover any `derived*` columns.

This logic is doing burst-rollup and should be targeting `BURST_COL`.  It was ok before because `BURST_COL` was a subset of `Metric::Capture.capture_cols`, but now it is not.